packages/cozy-sharing/src/components/ShareByLink.jsx

```diff
@@ -66,47 +77,163 @@ class ShareByLink extends React.Component {
     }
   }
 
+  toggleMenu = () => {
+    this.setState(state => ({ ...state, menuIsOpen: !state.menuIsOpen }))
+  }
+
   render() {
     const t = this.context.t
```

Comment:
I know this is not the scope of the PR but removing the context access can be a great improvement when working on an old file